test/ngResource/resourceSpec.js

```diff
@@ -753,6 +753,26 @@ describe('basic usage', function() {
     expect(json).toEqual({id: 123, number: '9876', $myProp: 'still here'});
   });
 
+  it('should not include $cancelRequest when resource is toJson\'ed', function() {
+    $httpBackend.whenGET('/CreditCard').respond({});
+
+    var CreditCard = $resource('/CreditCard', {}, {
+      get: {
+        method: 'GET',
+        cancellable: true
+      }
+    });
+
+    var creditCard = CreditCard.get();
+
+    expect(creditCard.$cancelRequest).toBeDefined();
+
+    $httpBackend.flush();
```

Comment:
Yes, we need to make sure It was there before remove it. same as the test next to it (for `$promise` and `$resolved`).